docs/source/api_reference.rst

```diff
@@ -21,8 +21,12 @@ Additional Utilities
 - **Utils** (`czbenchmarks.utils`):  
    Contains utility functions and helpers used across the package. See the full documentation: :doc:`./autoapi/czbenchmarks/utils/index`.
 
-- **CLI** (`czbenchmarks.cli`):  
-   Command-line interface for interacting with the cz-benchmarks package. See the full documentation: :doc:`czbenchmarks.cli <./autoapi/czbenchmarks/cli/cli/index>`.
+- **File Utils** (`czbenchmarks.file_utils`):  
+   Provides file handling utilities, such as loading datasets from various formats. See the full documentation: :doc:`./autoapi/czbenchmarks/file_utils/index`.
+
+- **Cache** (`czbenchmarks.file_cache`):  
+   Implements caching mechanisms to speed up repeated evaluations by storing results. See the full documentation: :doc:`./autoapi/czbenchmarks/file_cache/index`.
```

Comment:
The file_cache module is not used. We copied out this code from cli_run.py but never called it, and equivalent functions remain in cli_run.py. So we can probably remove this module and the documentation here.